mindsdb/utilities/functions.py

```diff
@@ -17,9 +17,9 @@ def cast_row_types(row, field_types):
     keys = [x for x in row.keys() if x in field_types]
     for key in keys:
         t = field_types[key]
-        if t == 'Timestamp' and row[key] is not None:
+        if t == 'Timestamp' and (isinstance(row[key], int) or isinstance(row[key], float)):
```

Comment:
`isinstance(row[key], int) or isinstance(row[key], float)` simplifies to `isinstance(row[key], (int, float))`